consensus/state.go

```diff
@@ -1020,35 +1070,40 @@ func (cs *State) enterPropose(height int64, round int32) {
 
 	// Nothing more to do if we're not a validator
 	if cs.privValidator == nil {
-		logger.Debug("This node is not a validator")
+		logger.Debug("node is not a validator")
 		return
 	}
-	logger.Debug("This node is a validator")
+
+	logger.Debug("node is a validator")
 
 	if cs.privValidatorPubKey == nil {
 		// If this node is a validator & proposer in the current round, it will
 		// miss the opportunity to create a block.
-		logger.Error(fmt.Sprintf("enterPropose: %v", errPubKeyIsNotSet))
+		logger.Error("propose step; empty priv validator public key", "err", errPubKeyIsNotSet)
 		return
 	}
+
 	address := cs.privValidatorPubKey.Address()
 
 	// if not a validator, we're done
 	if !cs.Validators.HasAddress(address) {
-		logger.Debug("This node is not a validator", "addr", address, "vals", cs.Validators)
+		logger.Debug("node is not a validator", "addr", address, "vals", cs.Validators)
 		return
 	}
 
 	if cs.isProposer(address) {
-		logger.Debug("enterPropose: our turn to propose",
+		logger.Debug(
+			"propose step; our turn to propose",
 			"proposer", address,
-			"privValidator", cs.privValidator,
+			"priv_validator", cs.privValidator,
 		)
+
 		cs.decideProposal(height, round)
 	} else {
-		logger.Debug("enterPropose: not our turn to propose",
+		logger.Debug(
+			"propose step; not our turn to propose",
 			"proposer", cs.Validators.GetProposer().Address,
-			"privValidator", cs.privValidator,
+			"priv_validator", cs.privValidator,
```

Comment:
This was the issue from yesterday right. Do you think it might pay to instead print `cs.privValidatorPubKey.Address()` or simply `cs.privValidatorPubKey`

Reply:
Good catch! Let me make a follow up PR :)

Reply:
https://github.com/tendermint/tendermint/pull/6144